.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -0,0 +1,46 @@
+<!--- STOP! Before you open an issue please search this repository's issues to see if it has already been reported. This helps reduce duplicate issues from being created. -->
+
+**Short description of the issue**:
+
+*description here*
+
+**Expected outcome**:
+
+*what you expect to happen goes here*
+
+**Actual result**:
+
+*what actually happens goes here*
+
+**Self contained code example that reproduces the issue**:
+
+```
+  add your code in here
+```
+
+**ACP Extension version used:**
+
+*version goes here (list all the extension versions if using multiple)*
+
+- iOS: Find these in the Podfile.lock file generated for your application
+- Android: Run the following command with gradle: *./gradlew :app:dependencyInsight* and extract the com.adobe.marketing.mobile extension versions
+
+**Platform:**
```

Comment:
We should ask for the platform and the version. E.g. iOS 10.1, and maybe Xcode/Android studio version.